commit bff75369888f566015f361d3a7775d3a50274662 (refs/changes/42/5242/2)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2018-12-14T11:27:41-08:00 (10 months ago)
    
    TRITON-1043 VMAPI workflow function putVm should use job.params rather than job.params.payload
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/lib/workflows/job-common.js b/lib/workflows/job-common.js
index 5b9f560..4764c16 100644
--- a/lib/workflows/job-common.js
+++ b/lib/workflows/job-common.js
@@ -304,7 +304,7 @@ function pollTask(job, cb) {
 function putVm(job, cb) {
     var vmapi;
 
-    if (job.params.payload.do_not_inventory) {
+    if (job.params.do_not_inventory) {
         cb(null,
             'VM has do_not_inventory set, no need to putVm to VMAPI');
         return;
